nixos/modules/services/mail/opendkim.nix

```diff
@@ -57,9 +59,13 @@ in {
         '';
       };
 
-      keyFile = mkOption {
+      keyPath = mkOption {
```

Comment:
We should add a `mkRenamedOptionModule` for this change.

Reply:
ok, will fix that, thanks for pointing out!

Reply:
fix landed, as you wished.